MANIFEST.in

```diff
@@ -0,0 +1,37 @@
+recursive-include unionml/templates *
+
+include README.md
+include src/pythae/py.typed  # marker file for PEP 561
+
+include CHANGELOG.md
+include changelog.md
+include LICENSE
+include LICENSE.txt
+include CITATION.cff
+include *.cff  # citation info
+
+include MANIFEST.in
+include src/pythae/pyproject.toml
+include setup.py
+include setup.cfg
+include tox.ini
+
+include requirements.txt
+
+
+recursive-exclude tests *
+recursive-exclude docs *
+recursive-exclude examples *
+recursive-exclude site *
```

Comment:
No such file in repo

Reply:
If you build documentation with MkDocs, you will have a local folder called `site`. This is typically ignored by git. But if you create a package locally and push it to PyPI, then this line is necessary to skip including the site folder which is not supposed to be part of the source. 

It's just a fire-and-forget solution.

Reply:
I use sphinx to build the doc and everything will be handled by the `recursive-exclude docs *`

Reply:
Removed.